pylzy/lzy/api/v1/remote/runtime.py

```diff
@@ -244,15 +243,13 @@ async def __load_local_modules(self, module_paths: Iterable[str]) -> Sequence[st
         for local_module in module_paths:
 
             with tempfile.NamedTemporaryFile("rb") as archive:
-                with zipfile.ZipFile(archive.name, "w") as z:
-                    zip_module(local_module, z)
+                zip_module(local_module, archive)  # type: ignore[arg-type]
```

Comment:
You are definitely doing something wrong if you need to do `# type: ignore` at every usage